examples/rendering/demo.py

```diff
@@ -154,7 +154,7 @@ def main():
 
     ########################## forward + backward twice ##########################
     scene.reset()
-    horizon = 2000
+    horizon = 1
```

Comment:
What are you doing here? This will break the demo as it will open and close instantly with going the opportunity to the user to see anything...

Reply:
done, it was because rendering 2000 images take too long. Reduce it to 10 (takes 10 second.).